README.md

```diff
@@ -28,7 +28,7 @@ docker-compose up
 
 Create an example Transfer
 ```
-./examples/getting-started.sh
+go run examples\getting-started\main.go
```

Comment:
I think `\` slashes only work on windows. Can we use `/` - I'm pretty sure Go supports that on Windows

```
$ go run .\cmd\server 
go: finding module for package .cmdserver
package .cmdserver: malformed module path ".cmdserver": leading dot in path element
```

Reply:
Done